internal/metadataproviders/openshift/metadata.go

```diff
@@ -0,0 +1,180 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package openshift // import "github.com/open-telemetry/opentelemetry-collector-contrib/internal/metadataproviders/openshift"
```

Comment:
why this commented import?

Reply:
did it like in the other `metadataproviders` packages.
e.g.:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/f4849ac26cd4b7b9f734c3b40c82525db2ad23d3/internal/metadataproviders/system/metadata.go#L15
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/f4849ac26cd4b7b9f734c3b40c82525db2ad23d3/internal/metadataproviders/internal/goos.go#L15
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/f4849ac26cd4b7b9f734c3b40c82525db2ad23d3/internal/metadataproviders/docker/metadata.go#L15